app/Services/Forge/ForgeService.php

```diff
@@ -108,9 +108,11 @@ public function markSiteAsNewlyMade(): void
 
     public function generateDatabaseName(): string
     {
-        return Str::limit(
-            Str::slug($this->setting->branch, '_'),
-            64
-        );
+        $this->getFormattedDomainName();
```

Comment:
Also it would be great to extract the logic inside `GenerateDomainName@formatBranchName` to a new action so we can use it here as well, since we get a full domain name instead of the branch name by calling the `getFormattedDomainName`. e.g., "branch-name.veyoze.com" which is not acceptable for database name, etc...

Reply:
yeah... i think I was confusing the formatted branch name with formatted domain name... apologies.